scripts/debug_tools/prepare_analyzer_cmd.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+import argparse
+import re
+import failure_lib as lib
+import os
+
+
+def getFirstLineOfFile(fname):
+    content = ''
+    with open(fname) as f:
+        content = f.readlines()
```

Comment:
You could use `readline` to read only the first line: https://docs.python.org/2/library/stdtypes.html#file.readline
This way you do not need to read the whole file.